kurl_proxy/kustomize/overlays/dev/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kotsadm-tls
+type: Opaque
```

Comment:
We switched this to kubernetes.io/tls in a recent kurl release.  i can fix this up later if you want.